src/components/Editor/ClientCursor.ts

```diff
@@ -3,20 +3,22 @@ import invert from 'invert-color';
 const markerRemoveMap = new Map<string, number>();
 
 const duration = 0.2;
-const delay = 2;
-const SECOND = 1000;
+const delay = 1;
+const SECOND = 500;
```

Comment:
This variable seems to have been declared for use by raising the unit of a millisecond to a second. 
Is there any reason to change it to 500?

Reply:
@hackerwins It's my fault,,